docs/layouts/shortcodes/generated/rest_v1_dispatcher.html

```diff
@@ -1084,7 +1084,38 @@
     "jobs" : {
       "type" : "array",
       "items" : {
-        "type" : "any"
+        "type" : "object",
+        "id" : "urn:jsonschema:org:apache:flink:runtime:messages:webmonitor:JobDetails",
+        "properties" : {
+          "duration" : {
+            "type" : "integer"
+          },
+          "end-time" : {
+            "type" : "integer"
+          },
+          "jid" : {
+            "type" : "any"
+          },
+          "last-modification" : {
+            "type" : "integer"
+          },
+          "name" : {
+            "type" : "string"
+          },
+          "start-time" : {
+            "type" : "integer"
+          },
+          "state" : {
+            "type" : "string",
+            "enum" : [ "INITIALIZING", "CREATED", "RUNNING", "FAILING", "FAILED", "CANCELLING", "CANCELED", "FINISHED", "RESTARTING", "SUSPENDED", "RECONCILING" ]
+          },
+          "tasks" : {
+            "type" : "object",
+            "additionalProperties" : {
+              "type" : "integer"
+            }
+          }
```

Comment:
This is a bit of a short-cut, but the alternative would be hard-code a class with a field for each execution state (+ 1 additional field to the total count.
This is how the field looks like in practice:
```
{
"total":3,
"created":0,
"scheduled":0,
"deploying":0,
"running":0,
"finished":3,
"canceling":0,
"canceled":0,
"failed":0,
"reconciling":0,
"initializing":0
}
```